lib/listen/adapter.rb

```diff
@@ -24,7 +24,7 @@ def select(options = {})
         Polling
       rescue
         Listen.logger.warn format('Adapter: failed: %s:%s', $ERROR_POSITION.inspect,
-                           $ERROR_POSITION * "\n")
+                                  $ERROR_POSITION * "\n")
```

Comment:
Style are personal preferences, but giving more spaces looks unnecessary to me.

Reply:
`rubocop` did that. I've stopped trying to predict its preference for spacing.

Reply:
Okay.